lazy_load_symbols/README.md

```diff
@@ -0,0 +1,16 @@
+Load symbols lazily
+===================
+
+Loads only the debugging symbols effectively used
+in the recording.
+
+To use:
+
+  1. start UDB with no parameters: udb
```

Comment:
Can you please use backtips around shell and UndoDB commands so they get rendered properly?

Reply:
yes I can